cli/repl.rs

```diff
@@ -94,7 +94,7 @@ impl Repl {
       .editor
       .readline(&prompt)
       .map(|line| {
-        self.editor.add_history_entry(line.as_ref());
+        self.editor.add_history_entry(line.clone());
```

Comment:
Hm I'm not sure what was happening on the other branch - I had to make that change to get it to compile locally... but if this is passing CI, I think it would be better left as is.

Reply:
Otherwise LGTM.

Reply:
I changed it to `&line` in the other branch and you pushed a commit to `line.clone()`.